website/content/docs/k8s/helm.mdx

```diff
@@ -251,14 +246,14 @@ Use these links to navigate to a particular top-level stanza.
 
         - `caCert` ((#v-global-secretsbackend-vault-connectinject-cacert)) - Configuration to the Vault Secret that Kubernetes uses on
           Kubernetes pod creation, deletion, and update, to get CA certificates
-          used issued from vault to send webhooks to the connect inject.
+          used issued from vault to send webhooks to the ConnectInject.
 
           - `secretName` ((#v-global-secretsbackend-vault-connectinject-cacert-secretname)) (`string: null`) - The Vault secret path that contains the CA certificate for
-            connect inject webhooks.
+            Connect Inject webhooks.
```

Comment:
One word above and below.
```suggestion
            ConnectInject webhooks.
```

Reply:
Hmm will think about these connect inject differences in a separate PR in consul-k8s later.